mastodon/mastodon-ios
# Delete CoreData records when user is blocked / muted from outside the App

# Rationale

As a user I want blocked users, which I have blocked from outside the App, to disappear from my timelines within the iOS App.

Resolves #626 

# How to test

* Log in to the iOS App
* Log in to Web (or another App)
* Mute / Block a user
  * Open the iOS App from Background / pull to refresh the timeline
  * Verify posts from the muted / blocked user have disappeared

@MainasuK / @zeitschlag could you please give this some QA? Thx!
